libs/causality/.eslintrc.json

```diff
@@ -5,6 +5,12 @@
     {
       "files": ["*.ts", "*.tsx", "*.js", "*.jsx"],
       "rules": {}
+    },
+    {
+      "files": ["**/LargeCausalIndividualChart.tsx"],
```

Comment:
I tried to limit the lines to 300 but I ran out of ideas and am at 326! Please let me know if possible to reduce lines, happy to update :)

Reply:
FWIW I'm fine with not being dogmatic about it and allowing exceptions. I would argue even 500 isn't a huge deal assuming the author tries to make it a logical small component without too much logic.